tests/x11regressions/remote_desktop/xdmcp_gdm.pm

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Remote Login: XDMCP with gdm and SLE-Classic configured
+# Maintainer: Chingkai <qkzhu@suse.com>
+# Tags: tc#1586203
+
+use strict;
+use base 'basetest';
+use base 'x11test';
+use testapi;
+use lockapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+
+    # Wait for supportserver if not yet ready
+    mutex_lock 'dhcp';
+    mutex_unlock 'dhcp';
+    mutex_lock 'xdmcp';
+
+    # Make sure the client gets the IP address and turn off firewall
+    x11_start_program 'xterm';
+    assert_screen 'xterm';
+    become_root;
+    assert_script_run 'dhclient';
+    assert_script_run 'systemctl -q is-active SuSEfirewall2 && systemctl disable SuSEfirewall2; systemctl stop SuSEfirewall2';
```

Comment:
why do you turn off the firewall? can instead the firewall be configured for the allowed service?